react/src/components/lablupTalkativotUI/LLMChatCard.tsx

```diff
@@ -160,12 +165,41 @@ const LLMChatCard: React.FC<LLMChatCardProps> = ({
     }
   }, [inputMessage, setInput]);
 
+  // If the `inputAttachment` prop exists, the `files` state has to follow it.
+  useEffect(() => {
+    if (!_.isUndefined(inputAttachment) && !_.isEqual(files, inputAttachment)) {
+      setFiles(inputAttachment);
+      setIsOpenAttachments(true);
+    }
+  }, [inputAttachment, files]);
+
+  const createDataTransferFiles = (files: AttachmentsProps['items']) => {
```

Comment:
Extract this function would be better. because it looks like independent subroutine function not require lexical scope

Reply:
@agatha197 I can't see any changes in here. I suggested that `createDataTransferFiles` should be placed to the out of the LLMChatCard, defined with function syntax

function createDataTransferFiles() {}